### DISCUSSION TITLE
Routing events using route transform

### DISCUSSION THREAD

**Discussion by amanmahajan26** [vectordotdev/vector#15220] [Q&A] [answered]
Hi,

According to the documentation: https://vector.dev/docs/reference/configuration/transforms/route/#route,  `If an event doesn’t match any route, it will be sent to the <transform_name>._unmatched output.`. Can we override this behaviour somehow? 

As in if the event does not match the previous routes, route it to this route. Just like the logical `if {} elseif {} else {}`, where `else` block catches all the rest of the events. The reason I am asking is because it gets very complicated to write the condition to match all the events which didn't match the previous routes.

What is a recommended way to handle this?

**Comment by bruceg** [comment, marked as answer]
There is no way to accomplish this currently without explicitly making the conditions mutually exclusive. We have [an open issue requesting this feature](https://github.com/vectordotdev/vector/issues/9100), but we have not prioritized it yet.
